app/services/stock/controllers/equityController.js

```diff
@@ -0,0 +1,63 @@
+const fetch = require('node-fetch');
+const Equity = require("../models/equityModel.js");
+require('dotenv').config();
+
+const CACHE_TIME = 1;
+
+const getStatistics = async (ticker) => {
```

Comment:
I think this should be included in the try block

Reply:
^The stuff below this

Reply:
yes, done